src/manifest_builder/manifest_builder.py

```diff
@@ -280,7 +280,7 @@ def populate_datasets(
         "--datasets_to_skip",
         nargs='+',
         type=str,
-        default=['releases', 'SEAAD', 'Zhuang-C57BL6J'],
+        default=['releases', 'SEA', 'Zhuang-C57BL6J'],
         help="Skip a given project for all directories that start with the "
              "given pattern. (e.g. SEAD will exclude all directories that "
```

Comment:
Should the example in the docstring be `SEAAD` (two As), since that is what we call the actual dataset?

Reply:
No. Pretty sure there are also directories with SEA-AD and we also want to skip those.